CitationStyleLanguageSettingsForm.php

```diff
@@ -12,65 +12,88 @@
  * @brief Form for site admins to modify Citation Style Language settings.
  */
 
+namespace APP\plugins\generic\citationStyleLanguage;
+
+use APP\core\Application;
+use APP\facades\Repo;
 use APP\notification\NotificationManager;
+use APP\template\TemplateManager;
 use PKP\form\Form;
-
+use PKP\form\validation\FormValidatorCSRF;
+use PKP\form\validation\FormValidatorPost;
 use PKP\notification\PKPNotification;
+use PKP\security\Role;
 
 class CitationStyleLanguageSettingsForm extends Form
 {
-    /** @var object $plugin */
-    public $plugin;
+    /** @var CitationStyleLanguagePlugin $plugin */
+    public CitationStyleLanguagePlugin $plugin;
 
     /**
      * Constructor
      *
+<<<<<<< HEAD:CitationStyleLanguageSettingsForm.inc.php
```

Comment:
There's some cruft here from the rebase.

Reply:
Sorry. It's reworked now.